jwst/regtest/test_niriss_sourcefind.py

```diff
@@ -24,6 +24,9 @@ def test_tweakreg_catalog_starfinder_alternatives(rtdata, starfinder):
             'minsep_fwhm': 2.5,
             'sigma_radius': 2.5,
         })
+    output_name = f"{stem}_{starfinder}_cat.ecsv"
+    catalog.write(output_name, format='ascii')
```

Comment:
Not sure if it makes any difference, but it looks like the format is usually 'ascii.ecsv' for writing these tables.

Reply:
It looks like the suffix is sufficient for (and perhaps overrides?) the format: https://docs.astropy.org/en/stable/io/ascii/ecsv.html

I'm not sure if that suggests I should remove the format entirely or if it's fine as is.

Reply:
Are both of your test cases with filename that includes the ecsv suffix? I'll add the extra formatting, I just read the documentation to state that the explicit filename suffix was sufficient for the intended behavior.